src/normalize.css

```diff
@@ -0,0 +1,4 @@
+html {
+  box-sizing: border-box;
```

Comment:
Placing it here, you apply `box-sizing` only for `html` but not for the rest of your elements